### PR TITLE
folly 2017.03.06.00

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
In file included from ClockGettimeWrappers.cpp:17:
In file included from ./../folly/ClockGettimeWrappers.h:19:
./../folly/portability/Time.h:32:17: error: typedef redefinition with different types ('uint8_t' (aka 'unsigned char') vs 'enum clockid_t')
typedef uint8_t clockid_t;
                ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/include/time.h:171:3: note: previous definition is here
} clockid_t;
  ^
1 error generated.
```

File: Formula/folly.rb [hunk 1]
```diff
@@ -40,25 +39,12 @@ def install
...
-        inreplace ["Benchmark.cpp", "Benchmark.h"] do |s|
-          s.gsub! "clock_gettime(CLOCK_REALTIME",
-                  "clock_gettime((clockid_t)CLOCK_REALTIME"
-          s.gsub! "clock_getres(CLOCK_REALTIME",
-                  "clock_getres((clockid_t)CLOCK_REALTIME", false
-        end
-
         # Fix "candidate function not viable: no known conversion from
         # 'folly::detail::Clock' to 'clockid_t' for 1st argument"
         # See upstream PR mentioned above
```
Comment (fxcoudert) [inline Formula/folly.rb]: `See upstream PR from 2 Oct 2016 facebook/folly#488` should be there instead of “mentioned above”

Comment (ilovezfs) [inline Formula/folly.rb]: It needs to be re-reported actually

**Comment by ilovezfs** [comment]
@fxcoudert re-reported https://github.com/facebook/folly/issues/557.

**Pull request closed without merging**